config/devices.yml

```diff
@@ -91,6 +91,15 @@ outside_heat:
     'ON':       0.9
   UT:
```

Comment:
Let's remove this entry since it adds confusion now that we have comm section. I mean the UT section